src/dualscreen.tsx

```diff
@@ -342,17 +367,23 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
           const ks: string = this._player.config.session.ks;
           if (!entryId) {
             this.logger.warn('Secondary entry id not found');
+            // subscribe on timed metadata events for image player
+            this._secondaryPlayerType = PlayerType.Image;
+            this.eventManager.listen(this.player, this.player.Event.TIMED_METADATA, this._onTimedMetadataLoaded);
+            this.eventManager.listen(this.player, 'metadata_added', this._onTimedMetadataAdded); // TODO: use enum
```

Comment:
needs to be enumed, and changed to `TIMED_METADATA_ADDED` | `timed_ metadata_added `